src/shared/utilities/download.ts

```diff
@@ -18,7 +18,6 @@ import { fsCommon } from '../../srcShared/fs'
 export class ExportResultArchiveStructure {
     static readonly PathToSummary = path.join('summary', 'summary.md')
     static readonly PathToDiffPatch = path.join('patch', 'diff.patch')
-    static readonly PathToSourceDir = 'sources'
```

Comment:
No longer using this anywhere. Also, the `download.ts` above was deleted since that whole file was a duplicate and not used anywhere.